packs/data/paizo-pregens.db/popcorn.json

```diff
@@ -1,77 +1,9 @@
 {
-    "_id": "LWsUZveKng8MhJrO",
+    "_id": "CLeXSMFf8xeRRf2t",
```

Comment:
```suggestion
    "_id": "LWsUZveKng8MhJrO",
```
We want to be really careful that IDs stay stable

Reply:
Ah, good call.